Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY . /code/
 
 # See from a previously published version to avoid pulling from Docker Hub (docker.io)
 # This version is only used to install the real version
-FROM ghcr.io/openzipkin/alpine:3.12.1 as install
+FROM ghcr.io/openzipkin/alpine:3.12.3 as install
```

Comment:
Should this be 3.14.2?

Reply:
Nevermind me. I actually read the comment above it now :)

Reply:
Yep and I reverted back to 3.12.1 as we have a deployment issue for arm. Isn't solved. Not sure how this exactly fails...